presto-client/src/main/java/io/prestosql/client/ClientCapabilities.java

```diff
@@ -15,5 +15,6 @@
 
 public enum ClientCapabilities
 {
-    PATH;
+    PATH,
+    PARAMETRIC_DATETIME;
```

Comment:
Maybe add a comment here.  I expect we will use this flag for timestamp and time with and without TZ

Reply:
I'm not sure how all clients work, but they might not expect the parametric timestamp in the returned type list of the results metadata. To address that you would change `io.prestosql.server.protocol.Query#toClientTypeSignature`

Reply:
That's a good question. In client type signature, they appear as parameters in a separate structure. I wonder whether clients validate that there are no parameters for timestamp or just match on the name and ignore the parameters if they are present.

Reply:
@electrum, any thoughts?

Reply:
I added a check for capabilities in toClientTypeSignature.